### DISCUSSION TITLE
How to filter/delete directory with underscore in the name like this: __tests__ ?

### DISCUSSION THREAD

**Discussion by flaviojoaofelix** [newren/git-filter-repo#394] [Q&A] [answered]
Hello!
I'm using the filter-repo and I can't get it to exclude/filter a directory named `__tests__` (two underscores before and after the name)

I've tried a few ways but no success! Could anyone help me with this?
I want to delete/filter this directory and all the files inside it.

I appreciate any help!

**Comment by newren** [comment]
Where is the __tests__ directory relative to the top of your git repository?  (In other words if you cd into your __tests__ directory and run `git rev-parse --show-prefix`, what is the output?)

Are you using --invert-paths?

What did you try?

I'd expect this to look something like
    git filter-repo --path some/sub/directory/__tests__ --invert-paths
but I'm just guessing at what the path might be to your directory from the top of the git repository.

**Reply by flaviojoaofelix** [reply]
Hello Newren,
Thanks for answering!

> Where is the tests directory relative to the top of your git repository?

_It is in the root folder of the repository_

> In other words if you cd into your tests directory and run git rev-parse --show-prefix, what is the output?

`__tests__/`

> Are you using --invert-paths?

_Yes, I've tested it in a few ways, including --invert-paths_

> I'd expect this to look something like: git filter-repo --path some/sub/directory/tests --invert-paths

originally test like this, but it didn't work:
```
--path __tests__/req17-deleteUsers.test.js \
--path __tests__/req18-searchPost.test.js \
--path __tests__ \
--invert-paths --force
```
so i tried some other ways but without success too
ex:
```
--path '__tests__'/req17-deleteUsers.test.js \
--path '__tests__' \
--invert-paths --force
OR
--path '__tests__/req17-deleteUsers.test.js' \
--invert-paths --force
OR
--path __tests__/req17-deleteUsers.test.js --invert-paths
--path __tests__ --invert-paths
```

Do you know where I could be going wrong?
Thank you so much for trying to help!

**Reply by flaviojoaofelix** [reply, marked as answer]
I ended up missing a simple error and got the solution now. Just the slash at the end of the directory.

I was trying like this:
```
--path __tests__ \
--invert-paths --force
```

And it was like this:
```
--path __tests__/ \
--invert-paths --force
```

Sorry to bother you with something so simple, but I really appreciate your attention!
